packages/devtools/lib/src/timeline/frames_bar_chart.dart

```diff
@@ -36,34 +34,24 @@ class FramesBarChart extends CoreElement {
         add(frameUIgraph);
       }
 
-      print(">>>>onFrameAdded Event");
       frameUIgraph.process(timelineController, frame);
-/*
-      final CoreElement frameUI = FrameBar(this, frame);
-      if (element.children.isEmpty) {
-        add(frameUI);
-      } else {
-        if (element.children.length >= maxFrames) {
-          element.children.removeLast();
-        }
-        element.children.insert(0, frameUI.element);
-      }
-*/
     });
   }
 
-  static const int chartHeight = 500;
+  static const int chartHeight = 300;
 
   static const int maxFrames = 500;
   static const topPadding = 2;
 
+  PlotlyDivGraph frameUIgraph;
   FrameBar selectedFrame;
 
   final StreamController<TimelineFrame> _selectedFrameController =
       StreamController<TimelineFrame>.broadcast();
 
   Stream<TimelineFrame> get onSelectedFrame => _selectedFrameController.stream;
 
+  // TODO(terry): Need to understand from Kenzie how much of setSelected I need from below???
```

Comment:
@kenzieschmoll can you follow up on what we need to do for setSelected?

Reply:
Ah most of that was to select and unselect the actual bars in the chart, so we shouldn't need that. We may want to keep the selectedFrame around to make sure we aren't adding the same frame to the stream via `_selectedFrameController.add(frame)`. 

So the method body would look like
```
if (selectedFrame == frame) {
  return;
}
_selectedFrameController.add(frame);
```

Reply:
Done. Thanks.